app/controllers/eligibility.ts

```diff
@@ -102,12 +102,31 @@ function getIneligible(req: Request, res: Response, next: NextFunction) {
   }
 }
 
+function getEligibleStart(req: Request, res: Response, next: NextFunction) {
+  try {
+    res.render('eligibility/eligibility-start.njk', {
+    });
+  } catch (e) {
+    next(e);
+  }
+}
+
+function postEligibleStart(req: Request, res: Response, next: NextFunction) {
+  try {
+    res.redirect(paths.start);
+  } catch (e) {
+    next(e);
+  }
+}
+
```

Comment:
I am confused here, I thought we should refactor the original start page to include a link to the 'eligible questions' and another link to 'login' in case the appellant has already an account. If that is the case we dont need another get/post method for this start page but just some refactoring of the original start page. This seems to me you are creating a new start page? also, the template you are trying to render here doesn't exist?